blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -781,37 +394,48 @@ function buildHeaderRow(category, headerType, isInactive) {
     return headerRow;
 }
 
-function buildTableRow(campaignJson, createHidden) {
-    //console.log(campaignJson);
+function buildTableRow(deliverableJson, kpi, createHidden) {
     const dataRow = document.createElement('div');
     dataRow.classList.add('row', 'datarow');
     if (createHidden) dataRow.classList.add('inactive');
+    const status = (deliverableJson.deliverableStatusUpdate == null) ? "??" : deliverableJson.deliverableStatusUpdate + "%";
```

Comment:
instead of ?? can you add Not Available?

Reply:
Done. :)